Documentation/install/upgrade.rst

```diff
@@ -181,7 +184,7 @@ Kubernetes resources are updated accordingly to version you are upgrading to:
    ``install/kubernetes/cilium/values.yaml`` and use it to regenerate the YAML
```

Comment:
What about the `helm upgrade cilium |CHART_RELEASE| --namespace=kube-system` on line 178?

Reply:
I hadn't tested it so didn't change it yet. I just tested it and will push it with the next rebase.